public/js/controllers/pagenavigation_controller.js

```diff
@@ -4,14 +4,12 @@ import Url from 'url-parse'
 
 export default class extends Controller {
   static get targets () {
-    return ['pagesize', 'votestatus', 'listview']
+    return ['pagesize', 'votestatus', 'listview', 'pageage']
```

Comment:
` 'age'` should just work fine here.

Reply:
Actually `datastatus` fits the `Newer` and `older` statuses much more better than `age`.

Reply:
this will no longer be needed as it's going back to being buttons